src/server/game/AuctionHouseBot/AuctionHouseBot.cpp

```diff
@@ -147,7 +147,12 @@ void AuctionBotConfig::GetConfigFromFile()
     SetConfigMax(CONFIG_AHBOT_NEUTRAL_ITEM_AMOUNT_RATIO, "AuctionHouseBot.Neutral.Items.Amount.Ratio", 100, 10000);
 
     SetAHBotIncludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceIncludeItems", ""));
-    SetAHBotExcludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceExcludeItems", ""));
+    std::string ahBotExcludes = sConfigMgr->GetStringDefault("AuctionHouseBot.forceExcludeItems", "");
+    if (!ahBotExcludes.empty())
+    {
+        TC_LOG_WARN("server.loading", "AuctionHouseBot.forceExcludeItems will be deleted soon. Please, use DisableMgr instead.");
```

Comment:
Minor note about warning log - end user doesn't neccessarily know what `DisableMgr` is but knows `AuctionHouseBot.forceExcludeItems` and knows `disables` db table (mention that instead)